### DISCUSSION TITLE
http://surenpi.com/ 博客访问不通

### DISCUSSION THREAD

**Discussion by crossroadmeet** [LinuxSuRen/phoenix.webui.framework#96] [Q&A]
你好，http://surenpi.com/ 博客访问不通，请问还有什么渠道可以查看 phoenix.webui.framework相关使用文档说明？包括配置和注解使用部分

**Comment by LinuxSuRen** [comment]
很感谢对该项目感兴趣，个人服务器没有费用，就停了。这里有部分文档 https://github.com/LinuxSuRen/phoenix.webui.framework/tree/master/docs

这里有 demo https://github.com/LinuxSuRen/phoenix.webui.framework.demo
